src/client/api.ts

```diff
@@ -757,6 +782,37 @@ export interface GetStakingContextRequest {
      */
     'options': { [key: string]: string; };
 }
+/**
+ * The balance of an asset onchain at a particular block
+ * @export
+ * @interface HistoricalBalance
+ */
+export interface HistoricalBalance {
```

Comment:
Is this new object part of the PR? Or unanticipated generated changes that came along with the proposed changes? If this was intentional, then `HistoricalBalance` might be too vague for what we are adding here. Perhaps it'd be more accurate to call this `HistoricalStakingBalance`?

Reply:
> unanticipated generated changes that came along with the proposed changes

This is it.